schemas/collaborations.yml

```diff
@@ -46,6 +47,7 @@ department: "enum(
             'computational-psychiatry',
             'sociology',
             'ecology',
+            'ecology-and-evolutionary-biology',
```

Comment:
I'm not sure this is quite right – it's what they have on the group website, but the Brown department is the "[Department of Ecology, Evolution and Organismal Biology](https://www.brown.edu/academics/ecology-and-evolutionary-biology/)". Maybe just leave "ecology"? 
Alternatively, we could update all the "department of ecology" tags.

Reply:
Hmm... definitely looks like there isn't just an "ecology" department. I'll ask Beth and see if she has any insight!